python-sdk/src/forta_agent/forta_graphql.py

```diff
@@ -27,40 +30,58 @@ def get_query(self):
         query = """
           query($input: AlertsInput) {
             alerts(input: $input) {
-              alerts {
-                  alertId
-                  addresses
-                  hash
-                  contracts {
-                      address
-                      name
-                      projectId
-                  }
-                  createdAt
-                  description
-                  findingType
-                  name
-                  projects {
-                      id
-                  }
-                  protocol
-                  metadata
-                  scanNodeCount
-                  severity
-                  source {
-                      transactionHash
-                      bot {
-                          id
-                      }
-                  }
-              }
-              pageInfo {
-                hasNextPage
-                endCursor {
-                  blockNumber
-                  alertId
+                alerts {
+                    alertId
+                    addresses
+                    contracts {
+                        name
+                        projectId
+                    }
```

Comment:
removing the `address` field could break existing bots that may be using the field